callisto/notification/models.py

```diff
@@ -64,7 +64,7 @@ def send(self, to, from_email, context=None):
         email.send()
 
     def add_site_from_site_id(self):
-        if getattr(settings, 'SITE_ID', None):
+        if not self.sites.count() and getattr(settings, 'SITE_ID', None):
```

Comment:
only adds the SITE_ID to the current email if the current email doesn't have any sites